src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -36,22 +41,32 @@ public JsonObject extendedLoan(Loan loan) {
       additionalBorrowerProperties(extendedRepresentation, loan.getUser());
     }else{
       //When there is no user, it means that the loan has been anonymized
-      extendedRepresentation.remove(LoanProperties.BORROWER);
+      extendedRepresentation.remove(BORROWER);
     }
 
-    if (loan.getLoanPolicy() != null) {
-      additionalLoanPolicyProperties(extendedRepresentation, loan.getLoanPolicy());
-    } else {
-      extendedRepresentation.remove(LoanProperties.LOAN_POLICY);
-    }
+    addPolicy(extendedRepresentation, loan.getLoanPolicy(), LOAN_POLICY);
+
```

Comment:
Personally, I might remove these blank lines, unless there is a need to clearly separate each policy operation.

Reply:
Fixed